app/domain/authentication/constraints/permitted_constraint.rb

```diff
@@ -0,0 +1,26 @@
+require 'command_class'
+
+module Authentication
+  module Constraints
+
+    PermittedConstraint = CommandClass.new(
```

Comment:
please call command classes with the action it performs. the name `PermittedConstraint` indicates a value object